examples/hf/pippy_t5.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 4 pippy_t5.py
+
+
+import argparse
+import os
+
+import torch
+import torch.distributed as dist
+
+from pippy.IR import Pipe, PipeSplitWrapper, annotate_split_points
+from pippy.PipelineStage import PipelineStage
+
+from transformers import T5ForConditionalGeneration, T5Config
+
+from hf_utils import generate_inputs_for_model, get_number_of_params
+
+
+def add_split_points(t5, nranks):
+    # Number of encoder layers: t5.config.num_layers
+    # Number of decoder layers: t5.config.num_decoder_layers
+    # 6 encoder layers, 6 decoder layers, 12 layers in total
+    total_layers = t5.config.num_layers + t5.config.num_decoder_layers
+    layers_per_rank = (total_layers + nranks - 1) // nranks
```

Comment:
what is this equation trying to express, is it the same thing as `math.ceil(total_layers / nranks)`?

Reply:
yes, ceil.